modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/ProcessResourceImpl.java

```diff
@@ -88,6 +95,51 @@ public Process getProcess(Long processId) throws Exception {
 		);
 	}
 
+	@Override
+	public Date getProcessLastSLACheckDate(Long processId) throws Exception {
+		SearchSearchRequest searchSearchRequest = new SearchSearchRequest();
+
+		searchSearchRequest.addSorts(
+			_sorts.field("lastCheckDate", SortOrder.DESC));
+
+		searchSearchRequest.setIndexNames(
+			_slaInstanceResultWorkflowMetricsIndexNameBuilder.getIndexName(
+				contextCompany.getCompanyId()));
+
+		BooleanQuery booleanQuery = _queries.booleanQuery();
+
+		BooleanQuery filterBooleanQuery = _queries.booleanQuery();
+
+		booleanQuery.addFilterQueryClauses(
+			filterBooleanQuery.addMustQueryClauses(
```

Comment:
Hey @MarinhoFeliphe, please, inline the `booleanQuery.addFilterQueryClauses(...)` into the `searchSearchRequest.setQuery(...)`.